### PR TITLE
Rename all meaningful occurrences of docker-py to docker-sdk-python

### PR DESCRIPTION
**Pull request by shin-**
## DO NOT MERGE

This should only be merged at release time, and once the GitHub repo has been renamed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Contributing guidelines
 
 See the [Docker contributing guidelines](https://github.com/docker/docker/blob/master/CONTRIBUTING.md).
-The following is specific to docker-py.
+The following is specific to the Docker SDK for Python.
```
Comment (bfirsh) [inline CONTRIBUTING.md]: It's a proper noun, so should be just "Docker SDK for Python".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -87,19 +88,18 @@ to reach out and ask questions. We will do our best to answer and help out.
 
 ## Development environment
 
-If you're looking contribute to docker-py but are new to the project or Python,
-here are the steps to get you started.
+If you're looking contribute to the Docker SDK for Python but are new to the
```
Comment (bfirsh) [inline CONTRIBUTING.md]: same - s/the//

**Comment by shin-** [comment]
I have done a partial revert of my previous changes. Moving the GitHub repo and docs is too disruptive right now and we need to move forward with the release. There will still be time to make those renames after the dust has settled a bit.

**Comment by TomasTomecek** [comment]
So the project will be renamed, is that correct? (I'm assuming we will need to do a rename in downstream distributions)

**Pull request merged**